eZ/Publish/Core/settings/search_engines/solr/sort_clause_visitors.yml

```diff
@@ -66,7 +66,7 @@ services:
     ezpublish.search.solr.content.sort_clause_visitor.map_location_distance:
         class: %ezpublish.search.solr.content.sort_clause_visitor.map_location_distance.class%
         arguments:
-            - @ezpublish.search.solr.content.field_map
+            - @ezpublish.search.field_name_resolver
```

Comment:
Best to fix it while it's hot :)
Updated all common service definitions in last commits (backports again).